graylog2-web-interface/src/components/search/MessageDetail.jsx

```diff
@@ -97,12 +97,19 @@ const MessageDetail = React.createClass({
       if (!streamList) {
         streamList = [];
       }
-      streamList.push(
-        <LinkContainer key={stream.id}
-                       to={Routes.stream_edit_example(stream.id, this.props.message.index, this.props.message.id)}>
-          <MenuItem>{stream.title}</MenuItem>
-        </LinkContainer>
-      );
+      if (stream.is_default) {
+        streamList.push(
+          <MenuItem key={stream.id} disabled={true} title="Cannot test against the default stream">{stream.title}</MenuItem>
```

Comment:
Minor nitpick: this breaks the [`react/jsx-boolean-value`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) style rule. You can just type `disabled` and the value passed to the component will be true, no need to assign the value